src/redux/reducer/order.reducer.js

```diff
@@ -0,0 +1,74 @@
+import { orderType } from "../actionType/actionTypes";
+
+const initialState = {
+  isLoading: false,
+  orderList: {},
```

Comment:
```suggestion
  orderList: [],
```
Array thay vì obj chứ nhỉ?